Documentation/distributions.md

```diff
@@ -146,7 +146,7 @@ upgrade manually.
 
 ### rpm-based 
 ```
-gpg --recv-key 18AD5014C99EF7E3BA5F6CE950BDD3E0FC8A365E
+gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-key 18AD5014C99EF7E3BA5F6CE950BDD3E0FC8A365E
```

Comment:
I agree with @squeed's [comment](https://github.com/rkt/rkt/issues/3919#issuecomment-379697969) that "we at the rkt project can't decide this".

I have no problem with us recommending a reasonable option, but I'd rather we make it clear that you should prefer any that your distro or you have already picked.

Perhaps having a comment to that effect would be enough, e.g.

```
# Note: the example keyserver argument should be omitted if you already have a keyserver configured or you do not trust the SKS Keyservers
```